packages-exp/performance-exp/src/index.ts

```diff
@@ -50,7 +50,7 @@ import { getModularInstance } from '@firebase/util';
 const DEFAULT_ENTRY_NAME = '[DEFAULT]';
 
 /**
- * Returns a FirebasePerformance instance for the given app.
+ * Returns a {@link FirebasePerformance} instance for the given app.
```

Comment:
It seems like all the other libraries don't have "Firebase" in the name of their instance.  e.g., `FirebaseMessaging` is now just `Messaging`.  There's also just `Auth`, `Functions`, `RemoteConfig`, etc.  I'm sure that I'm just missing/misunderstanding something, but should FirePerf be consistent with the other libraries?

Reply:
It is prefixed because `Performance` collides with the global typing for the performance API in browser.
See go/js-v9-types-naming

Reply:
ah! yes.  makes sense :-)